sdk/servicebus/service-bus/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,248 @@
+# Troubleshooting Service Bus issues
+
+This troubleshooting guide covers failure investigation techniques, common errors in the Azure Service Bus JavaScript client library, and mitigation steps to resolve these errors.
+
+## ToC
```

Comment:
I don't see an actual TOC, is this a mistake?

Reply:
ah, I still need to fill in this.